lib/openstudio-standards/infiltration/infiltration.rb

```diff
@@ -0,0 +1,183 @@
+module OpenstudioStandards
+  # The Infiltration module provides methods create, modify, and get information about model infiltration
+  module Infiltration
+    # @!group Calculations
+
+    # Convert an infiltration rate at a given pressure to another pressure (typically lower)
+    # Method from Gowri, Krishnan, Winiarski, David W, and Jarnagin, Ronald E. Infiltration modeling guidelines for commercial building energy analysis. United States: N. p., 2009. Web. doi:10.2172/968203.
+    #
+    # @param infiltration_rate [Double] initial infiltration rate
+    # @param initial_pressure [Double] pressure difference at which initial infiltration rate was determined, typically 75 Pa
+    # @param final_pressure [Double] desired pressure difference to adjust infiltration rate, typically 4 Pa
+    # @param infiltration_coefficient [Double] infiltration coeffiecient
+    # @return [Double] adjusted infiltration rate in the same units as infiltration_rate
+    def self.adjust_infiltration_to_new_pressure(infiltration_rate,
+                                                 initial_pressure: 75.0,
+                                                 final_pressure: 4.0,
+                                                 infiltration_coefficient: 0.65)
+      adjusted_infiltration_rate = infiltration_rate * (final_pressure / initial_pressure)**infiltration_coefficient
+
+      return adjusted_infiltration_rate
+    end
+
+    # Convert the infiltration rate to the pressures and conditions assumed in the PNNL prototype buildings
+    # Details described in Gowri, Krishnan, Winiarski, David W, and Jarnagin, Ronald E. Infiltration modeling guidelines for commercial building energy analysis. United States: N. p., 2009. Web. doi:10.2172/968203.
+    #
+    # @param infiltration_rate [Double] initial infiltration rate
+    # @param initial_pressure [Double] pressure difference at which initial infiltration rate was determined in Pa, default 75 Pa
+    # @return [Double] adjusted infiltration rate in the same units as infiltration_rate
+    def self.adjust_infiltration_to_prototype_building_conditions(infiltration_rate, initial_pressure: 75.0)
+      alpha = 0.22 # terrain adjustment factor for an urban environment, unitless
+      uh = 4.47 # wind speed, m/s
+      rho = 1.18 # air density, kg/m^3
+      cs = 0.1617 # positive surface pressure coefficient, unitless
+      n = 0.65 # infiltration coefficient, unitless
+
+      # Calculate the typical pressure - same for all building types
+      final_pressure_pa = 0.5 * cs * rho * uh**2
+      adjusted_infiltration_rate = (1.0 + alpha) * infiltration_rate * (final_pressure_pa / initial_pressure)**n
```

Comment:
why not call `adjust_infiltration_to_new_pressure` with these arguments?

Reply:
I could but this is clearer and shorter.  The alpha bit makes it a bit different, which is why we need a whole separate method for the prototype rates to begin with.